webviz_subsurface/_datainput/pvt_data.py

```diff
@@ -0,0 +1,112 @@
+########################################
+#
+#  Copyright (C) 2020-     Equinor ASA
+#
+########################################
+
+import pandas as pd
+from opm.io.ecl import EclFile
+
+from webviz_config.common_cache import CACHE
+from webviz_config.webviz_store import webvizstore
+
+from .opm_init_io import Oil, Gas, Water, DryGas
+
+try:
+    from .fmu_input import load_ensemble_set
+except ImportError:
+    from fmu_input import load_ensemble_set
+
+try:
+    import ecl2df
+except ImportError:
+    pass
```

Comment:
I guess `ecl2df` would fail in the same cases as `.opm_init_io` and `opm.io.ecl` (i.e. missing `opm` binaries)? Add them to same `try`/`except`?

That way we perhaps can offer Mac / Windows users still the csv-option (without failing on import).

Reply:
Yes, makes totally sense.